Tests/CommonMarkAttributedStringTests/CommonMarkAttributedStringTests.swift

```diff
@@ -66,4 +66,21 @@ final class CommonMarkAttributedStringTests: XCTestCase {
 
         XCTAssert(attributedString.string.starts(with: "Universal Declaration of Human Rights"))
     }
+
+    func testHeaderFont() throws {
+        let commonmark = "# Font family should not change"
+
+        #if canImport(UIKit)
+        typealias Font = UIFont
+        #elseif canImport(AppKit)
+        typealias Font = NSFont
+        #endif
```

Comment:
This is a personal preference, but could we handle UIKit / AppKit differences within conditional compilation instead? I know `UIFont` and `NSFont` (like `__Image` and a few other types) have, like 90% overlap in their API (indeed, 100% in this case), but I think the clarity of handling each in turn explicitly is worth the cost of duplication here — especially when we're using this pattern in the implementation.

Reply:
No problem! I had a sneaking suspicion that this was a little naughty, which is why I only defined the alias within this scope 😬 

This would certainly cause a world of pain aliased/overlapping APIs worked this way everywhere in the package.